### PR TITLE
Basic user details on Settings page

### PR DESCRIPTION
**Pull request by clusterwin**
demo

### CODE REVIEW

File: flixnchill/AppDelegate.m [hunk 1]
```diff
@@ -100,6 +100,8 @@ - (User *)setUpFacebookUser{
 		user.name = [data objectForKey:@"name"];
         user.email = [data objectForKey: @"email"];
         user.firstName = [data objectForKey: @"first_name"];
+		user.gender = [data objectForKey:@"gender"];
+		user.friends = @"44";
```
Comment (gprasant) [inline flixnchill/AppDelegate.m]: there should be a friends_count on the api you can use

**Comment by gprasant** [comment]
LGTM

File: flixnchill/SettingsViewController.m [hunk 2]
```diff
@@ -102,6 +110,51 @@ -(void)loginButtonClicked
...
+			[self.userDetails setValue:user.gender forKey:@"Gender"];
+		}
+		if(user.location){
+			[self.userDetails setValue:user.location forKey:@"Location"];
+		}
+		if(user.matches){
+			[self.userDetails setValue:[NSString stringWithFormat:@"%d",[user.matches count]] forKey:@"matches"];
+		}
+		if(user.friends){
+			[self.userDetails setValue:user.friends forKey:@"Friends on NF&C"];
```
Comment (greysongregory) [inline flixnchill/SettingsViewController.m]: what about "Friends flixin' & chillin' " or something? NF+C took me a second

Comment (clusterwin) [inline flixnchill/SettingsViewController.m]: I like it

File: flixnchill/AppDelegate.m (hunk 1 above)
Comment (clusterwin) [inline flixnchill/AppDelegate.m]: ah nice

**Pull request merged**